sycl/test/array_param/array-kernel-param-run.cpp

```diff
@@ -0,0 +1,251 @@
+// This test checks kernel execution with array kernel parameters.
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// XFAIL: *
```

Comment:
Can we split this test and move non-working (struct array member, I guess) case into another test file?
So we can enable runtime testing of non-wrapped arrays now.

Reply:
Done.